# Translation of Odoo Server.
# This file contains the translation of the following modules:
# 	* payment_dpo
# 
# Translators:
# Wil Odoo, 2025
# 
msgid ""
msgstr ""
"Project-Id-Version: Odoo Server saas~18.3\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2025-05-09 20:37+0000\n"
"PO-Revision-Date: 2025-05-17 15:02+0000\n"
"Last-Translator: Wil Odoo, 2025\n"
"Language-Team: Thai (https://app.transifex.com/odoo/teams/41243/th/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: \n"
"Language: th\n"
"Plural-Forms: nplurals=1; plural=0;\n"

#. module: payment_dpo
#. odoo-python
#: code:addons/payment_dpo/models/payment_transaction.py:0
msgid ""
"An error occurred during processing of your payment (code %(code)s: "
"%(explanation)s). Please try again."
msgstr ""

#. module: payment_dpo
#: model:ir.model.fields,field_description:payment_dpo.field_payment_provider__code
msgid "Code"
msgstr "โค้ด"

#. module: payment_dpo
#: model_terms:ir.ui.view,arch_db:payment_dpo.payment_provider_form
msgid "Company Token"
msgstr ""

#. module: payment_dpo
#: model:ir.model.fields.selection,name:payment_dpo.selection__payment_provider__code__dpo
msgid "DPO"
msgstr ""

#. module: payment_dpo
#: model:ir.model.fields,field_description:payment_dpo.field_payment_provider__dpo_company_token
msgid "DPO Company Token"
msgstr ""

#. module: payment_dpo
#: model:ir.model.fields,field_description:payment_dpo.field_payment_provider__dpo_service_ref
msgid "DPO Service ID"
msgstr ""

#. module: payment_dpo
#: model:ir.model.fields,field_description:payment_dpo.field_payment_provider__display_name
#: model:ir.model.fields,field_description:payment_dpo.field_payment_transaction__display_name
msgid "Display Name"
msgstr "แสดงชื่อ"

#. module: payment_dpo
#: model:ir.model.fields,field_description:payment_dpo.field_payment_provider__id
#: model:ir.model.fields,field_description:payment_dpo.field_payment_transaction__id
msgid "ID"
msgstr "ไอดี"

#. module: payment_dpo
#. odoo-python
#: code:addons/payment_dpo/models/payment_transaction.py:0
msgid "No transaction found matching reference %s."
msgstr "ไม่พบธุรกรรมที่ตรงกับการอ้างอิง %s"

#. module: payment_dpo
#: model:ir.model,name:payment_dpo.model_payment_provider
msgid "Payment Provider"
msgstr "ผู้ให้บริการชำระเงิน"

#. module: payment_dpo
#: model:ir.model,name:payment_dpo.model_payment_transaction
msgid "Payment Transaction"
msgstr "ธุรกรรมสำหรับการชำระเงิน"

#. module: payment_dpo
#: model_terms:ir.ui.view,arch_db:payment_dpo.payment_provider_form
msgid "Service ID"
msgstr ""

#. module: payment_dpo
#. odoo-python
#: code:addons/payment_dpo/models/payment_provider.py:0
msgid "The communication with the API failed."
msgstr "การสื่อสารกับ API ล้มเหลว"

#. module: payment_dpo
#: model:ir.model.fields,help:payment_dpo.field_payment_provider__code
msgid "The technical code of this payment provider."
msgstr "รหัสทางเทคนิคของผู้ให้บริการชำระเงินรายนี้"

#. module: payment_dpo
#. odoo-python
#: code:addons/payment_dpo/models/payment_transaction.py:0
msgid "Unknown status code: %s"
msgstr "ไม่ทราบรหัสสถานะ: %s"
